rdrf/rdrf/views/patient_view.py

```diff
@@ -539,22 +547,44 @@ def get_forms(self, request, registry_model, user, instance=None):
 
         return forms
 
-    def _section_hidden(self, user, registry, fieldlist):
+    def _exclude_hidden_fields(self, user, registry, fieldlist):
+        from rdrf.models.definition.models import DemographicFields
```

Comment:
Can we please move imports to the top of the file as a general rule?

Reply:
Updated.